src/ui/qml/LanguagePreferences.qml

```diff
@@ -0,0 +1,57 @@
+import QtQuick 2.0
+import QtQuick.Controls 1.0
+import QtQuick.Layouts 1.0
+import im.ricochet 1.0
+
+ColumnLayout {
+    anchors {
+        fill: parent
+        margins: 8
+    }
+
+    QtObject {
```

Comment:
It is not necessary but it should be a private property. It is not intended to be modified from outside.